### PR TITLE
Stars

### PR DESCRIPTION
**Pull request by blacktox96**
- [DEMO LINK](https://blacktox96.github.io/layout_stars/)
- [TEST REPORT LINK](https://blacktox96.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,10 +9,83 @@
...
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+        <div class="delimiter"></div>
+        <div class="stars__star"></div>
+      </div>
```
Comment (etojeDenys) [inline src/index.html:52]:
```suggestion
      <div class="stars stars--2">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
rewrite it without the additional elements

File: src/style.css [hunk 2]
```diff
@@ -1 +1,71 @@
...
+.stars--3 :nth-child(n + 6) {
+  background-image: url(/layout_stars/src/images/star.svg);
+}
+
+.stars--4 :nth-child(n + 8) {
+  background-image: url(/layout_stars/src/images/star.svg);
+}
+
+.stars--5 :nth-child(n + 10) {
+  background-image: url(/layout_stars/src/images/star.svg);
```
Comment (etojeDenys) [inline src/style.css:63]: combine it with comma

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (blacktox96) [reply src/index.html:52]: How i should indent them than?
(Its forbidden to use gap by task requirements)

File: src/style.css (hunk 2 above)
Comment (blacktox96) [reply src/style.css:63]: Done ✅

**Pull request closed without merging**